app/views/shopping_lists/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class ="container">
+<h1 class="display-5 text-center">Shopping List</h1>
+<table class="table table-striped table-hover mt-3">
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Quantity</th>
+      <th>Price</th>
+      <th>Total</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% total_price = 0 %>
+    <% @shopping_list.each do |food, details| %>
+      <% total_price += details[:price] * details[:quantity] %>
+      <tr>
+        <td><%= food %></td>
+        <td><%= details[:quantity] %></td>
+        <td><%= number_to_currency(details[:price]) %></td>
+        <td><%= number_to_currency(details[:price] * details[:quantity]) %></td>
+      </tr>
+    <% end %>
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/235281459-56e904de-6834-45c8-bebe-14adc313c259.png)

- [ ] Great work on adding the quantity and total price as suggested by the previous reviewer :+1: but I guess it is not showing the correct data :fearful: When I tried to add Garlic which has a price of 2 dollars and I have added quantity 13 so the total amount should be 26 which is shown in the table but on the top the total amount is different :fearful: Kindly fix it to make your app work as expected :+1:

**Kindly use the same scenario which I used because the value can vary :+1:**

I have also used the same scenario that the wireframe has :arrow_down: you can also use that :blush:

### Your App
![image](https://user-images.githubusercontent.com/93039359/235281811-4541d224-4add-412b-b113-adea46e8bda8.png)

### Given Wireframe
![image](https://user-images.githubusercontent.com/93039359/235281578-2cb58862-f72e-4867-8db7-579c6b7cd0aa.png)

Reply:
Ohhh thanks a lot for the feedback I just noticed what was happening, I literally was multiplying the quantity with the value of the item! 😅 
The issue has been resolved:
![14](https://user-images.githubusercontent.com/90111138/235284302-e55b379d-3540-46b1-aa9c-15d73e349f5c.JPG)